buildSrc/src/main/kotlin/Project.kt

```diff
@@ -1,7 +1,7 @@
 object AndroidVersions {
   const val minSdkVersion = 19
```

Comment:
Maybe bump version to 21 already here? Just because it's big spring cleanup? 🌸

Reply:
done